src/components/PlaceHolder.vue

```diff
@@ -0,0 +1,14 @@
+<template>
```

Comment:
Can you give a more meaningful name to this component? Like TodoListPlaceholder.vue?